src/main.js

```diff
@@ -167,6 +167,9 @@ function configureCommandlineSwitchesSync(cliArgs) {
 
 		// Force enable screen readers on Linux via this flag
 		SUPPORTED_ELECTRON_SWITCHES.push('force-renderer-accessibility');
+
+		// Specify ozone platform implementation to use.
+		SUPPORTED_ELECTRON_SWITCHES.push('ozone-platform');
```

Comment:
Since #154092, Visual Studio Code is using an Electron version that supports the `--ozone-platform-hint` flag. So it may be worth enabling this flag too (`ozone-platform-hint`) as part of this pull-request.

Reply:
@vially unfortunately we cannot support `ozone-platform-hint` from application code since the flag detection happens before application code starts its execution which would be in `ElectronBrowserMainParts::PreMainMessageLoopRun`